### PR TITLE
doc/architecture: edit "High Avail. Monitors"

### PR DESCRIPTION
**Pull request by zdover23**
Improve the sentence structure in the "High Availability Monitors" section of doc/architecture.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/architecture.rst [hunk 1]
```diff
@@ -179,21 +179,27 @@ information recording the overall health of the Ceph Storage Cluster.
...
-In a cluster of monitors, latency and other faults can cause one or more
-monitors to fall behind the current state of the cluster. For this reason, Ceph
-must have agreement among various monitor instances regarding the state of the
-cluster. Ceph always uses a majority of monitors (e.g., 1, 2:3, 3:5, 4:6, etc.)
-and the `Paxos`_ algorithm to establish a consensus among the monitors about the
-current state of the cluster.
-
-For details on configuring monitors, see the `Monitor Config Reference`_.
+A Ceph Client must contact a Ceph Monitor and obtain a current copy of the
+cluster map in order to be permitted to read data from or to write data to the
```
Comment (anthonyeleven) [inline doc/architecture.rst:183]: s/be permitted to//

File: doc/architecture.rst [hunk 2]
```diff
@@ -179,21 +179,27 @@ information recording the overall health of the Ceph Storage Cluster.
...
+Ceph cluster.
+
+It is possible for a Ceph cluster to function properly with only a single
+monitor in it, but a Ceph cluster that has only a single monitor in it has a
```
Comment (anthonyeleven) [inline doc/architecture.rst:187]: s/ in it//g

File: doc/architecture.rst [hunk 3]
```diff
@@ -179,21 +179,27 @@ information recording the overall health of the Ceph Storage Cluster.
...
+single point of failure: if the monitor goes down, Ceph clients will be unable
+to read data from or write data to the cluster.
+
+Ceph supports a cluster of monitors in order to increase reliability and fault
```
Comment (anthonyeleven) [inline doc/architecture.rst:191]: s/supports/leverages/

**Review by anthonyeleven — APPROVED** [review]

File: doc/architecture.rst (hunk 1 above)
Comment (zdover23) [reply doc/architecture.rst:183]: Good eye. Accepted.

File: doc/architecture.rst (hunk 2 above)
Comment (zdover23) [reply doc/architecture.rst:187]: Good eye here, too. Accepted.

File: doc/architecture.rst (hunk 3 above)
Comment (zdover23) [reply doc/architecture.rst:191]: This is that rarest of cases: the time when I think that "leveraged" is, in fact, the right word.

Accepted.

Comment (anthonyeleven) [reply doc/architecture.rst]: Ever since I read that "utilize" doesn't mean what I think it means I've shied away from it.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53451 - Reef backport
https://github.com/ceph/ceph/pull/53452 - Quincy backport

**Pull request merged**